server/index.js

```diff
@@ -25,6 +26,62 @@ app.get('/api/individual_reviews/:id' , (req, res) => {
   .sort({ date: -1 });
 });
 
+app.post('/api/reviews', (req, res) => {
+  let newReview = new Review(req.body);
+  newReview.save((err, review) => {
+    if (err) {
+      res.send(500).end();
+    } else {
+      res.status(200).end('Review Sent');
+    }
+  });
+});
+
+app.get('/api/reviews', (req, res) => {
+  let userName = req.body.user;
+  let stayDate = req.body.date;
+  let location = req.body.locationID;
+  Review.findOne({
+    user: userName,
+    date: stayDate,
+    locationID: location
+  }, (err, review) => {
+    if (err) {
+      res.send('An Error Occurred');
+    } else {
+      res.send(review);
+    }
+  });
+});
+
+app.put('/api/reviews/:id', (req, res) => {
+  let update = req.body;
+  let userName = req.body.user;
+  let stayDate = req.body.date;
+  let location = req.params.id;
+  Review.update({ user: userName, date: stayDate, locationID: location }, { $set: update }, (err, review) => {
+    if (err || review.nModified === 0) {
+      res.send('An Error Occurred.');
+    } else {
+      res.send('Review Updated');
+    }
+  });
+});
+
+app.delete('/api/reviews/:id', (req, res) => {
+  let userName = req.body.user;
+  let stayDate = req.body.date;
+  let location = req.params.id;
+  Review.remove({ user: userName, date: stayDate, locationID: location }, (err, review) => {
```

Comment:
You may want to make this line easier to read. 

Tip: you can use ES6 destructuring to achieve this. Example:

const { user, date } = req.body;

Review.remove({ user, date } ... )